moneyed/money.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
```

Comment:
yes I just moved the code to a new module,
the idea behind this was to keep backward compatibility with previous versions and also provide a way to load localization data, so let's take the code below as example:
```python
from moneyed import Money
sale_price_today = Money(amount='99.99', currency='USD')
print(sale_price_today.currency.name)
```
This code will call the loader with default system locale, so it will print `'US Dollar'` on my system. but user is also able to load the locale it wants, so
```python
from moneyed.money import Money
from moneyed.loader import load_currencies
load_currencies(locale='fr_FR')
sale_price_today = Money(amount='99.99', currency='USD')
print(sale_price_today.currency.name)
```
will print `'dollar des États-Unis'`.
It may look a bit messy but that's what came to my mind when I was writing this. but we can also think about a new api to support localization for currency data.